### PR TITLE
Document file permissions issue

### PR DESCRIPTION
**Pull request by yvanzo**
Based on feedback given in comment to the issue https://github.com/metabrainz/musicbrainz-docker/issues/263, this patch document when building Docker images is failing due to a host system hindering some file permissions.

### CODE REVIEW

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -109,3 +110,28 @@ sudo docker-compose restart indexer
...
+``` log
+Err:1 https://deb.nodesource.com/node_20.x nodistro InRelease
+  Unknown error executing apt-key
+[...]
+W: GPG error: https://deb.nodesource.com/node_20.x nodistro InRelease: Unknown error executing apt-key
+E: The repository 'https://deb.nodesource.com/node_20.x nodistro InRelease' is not signed.
+```
+
+This may happen if your system is hindering file permissions.
+You can find it out by adding `RUN ls -l file` commands in the
```
Comment (reosarevok) [inline TROUBLESHOOTING.md]: Maybe just:

```suggestion
You can find out by adding `RUN ls -l file` commands in the
```

File: TROUBLESHOOTING.md [hunk 2]
```diff
@@ -109,3 +110,28 @@ sudo docker-compose restart indexer
...
+Dockerfile.
+
+Solution:
+
+Configure your system to keep the file permissions defined in the Git repository
+and to preserve the permissions of the files copied through Docker.
+
+If it isn’t possible, for example with Unraid operating system,
```
Comment (reosarevok) [inline TROUBLESHOOTING.md]:
```suggestion
If it isn’t possible, for example with the Unraid operating system,
```
?

File: TROUBLESHOOTING.md [hunk 3]
```diff
@@ -109,3 +110,28 @@ sudo docker-compose restart indexer
...
+run additional `chmod` commands in the Dockerfile; See issue #263.
```
Comment (reosarevok) [inline TROUBLESHOOTING.md]: This doesn't create an automatic link to the issue in the preview, at least - should we link by hand?

File: TROUBLESHOOTING.md (hunk 1 above)
Comment (yvanzo) [reply TROUBLESHOOTING.md]: Applied as https://github.com/metabrainz/musicbrainz-docker/pull/266/commits/36a0878fb5d588bc30ca0a9f1fe519fe5bcbba88

File: TROUBLESHOOTING.md (hunk 2 above)
Comment (yvanzo) [reply TROUBLESHOOTING.md]: Thanks, applied as https://github.com/metabrainz/musicbrainz-docker/pull/266/commits/1162a205beac314fc5badab1417ce138d62a9dc2

File: TROUBLESHOOTING.md (hunk 3 above)
Comment (yvanzo) [reply TROUBLESHOOTING.md]: Good catch, fixed through https://github.com/metabrainz/musicbrainz-docker/pull/266/commits/0f2be8db32e9a5bd5fc22afb5287f7cdc427f014

**Review by reosarevok — APPROVED** [review]

**Pull request merged**